### PR TITLE
Make pretty URLs and default views work when using an Apache alias

### PR DESCRIPTION
**Pull request by jotaelesalinas**
To make pretty URLS work when using an Apache alias:
 - Added a `RewriteBase` directive to `public/.htaccess` (commented out by default)

To make links and forms in the default views work:
- Changed hardcoded absolute URLs for their relative counterparts, using `url()`.

### CODE REVIEW

**Comment by franzliedke** [comment]
+1 for using `url()`.

File: public/.htaccess [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
     RewriteEngine On
 
+	# Uncomment and set your actual path if you run your application using an alias
```
Comment (GrahamCampbell) [inline public/.htaccess]: incorrect indentation

Comment (jotaelesalinas) [inline public/.htaccess]: Fixed.

**Comment by GrahamCampbell** [comment]
Needs to be sent to the master.

**Comment by jotaelesalinas** [comment]
Ok, thanks. Pull request #3263 created in master.

**Pull request closed without merging**